README.md

```diff
@@ -31,8 +35,181 @@ const nuked = embed(app, {
   ],
 });
 
+// Rendering a scatter plot on the fly
 nuked.render({
+  type: 'scatterplot',
   element,
+  fields: ['Alpha', '=Sum(Expression1)', '=Sum(Expression2)', '=Sum(Expression3)'],
+  options: {
+    direction: 'ltr',
+    freeResize: true,
+  },
+  properties: {
+    color: { mode: 'byDimension' }, // overrides default properties
+  },
+});
+```
+
+## Options
+
+- direction - ltr/rtl
+- freeResize - in conjunction with snapshotData on layout, lets the chart ignore size set on snapshotData
+
+## Scatter plot plugins
+
+A plugin can be passed into a scatter plot to add or modify its capability
+or visual appearance.
+A plugin needs to be defined before it can be rendered together with the chart.
+
+```js
+// Step 1: define the plugin
+const pointPlugin = {
+  info: {
+    name: 'point-plugin',
+    type: 'component-definition',
+  },
+  fn: ({ layout, keys }) => {
+    const componentDefinition = {
+      key: keys.components.point,
+      type: 'point',
+      settings: {
+        strokeWidth: '2px',
+        stroke: 'black',
+        size: (d) => getSizeInLogarithmScale(d, layout),
+        fill: (d) => getColorBasedOnMedian(d),
+      },
+    };
+    return componentDefinition;
+  },
+};
+
+// Step 2: passing the plugin definition into the render function
+embed.render({
```

Comment:
maybe change `embed` to `nuked` to be consistent with the code in Usage

Reply:
good point Bao, updated! ✅